homeassistant/components/wemo/binary_sensor.py

```diff
@@ -1,22 +1,20 @@
 """Support for WeMo binary sensors."""
-import asyncio
 
 from pywemo import Insight, Maker, StandbyState
 
 from homeassistant.components.binary_sensor import BinarySensorEntity
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
-from .const import DOMAIN as WEMO_DOMAIN
+from . import async_wemo_dispatcher_connect
 from .entity import WemoBinaryStateEntity, WemoEntity
 from .wemo_device import DeviceCoordinator
 
 
 async def async_setup_entry(
     hass: HomeAssistant,
-    config_entry: ConfigEntry,
+    unused_config_entry: ConfigEntry,
```

Comment:
We don't usually use this pattern, and our linters don't complain about unused parameters.
It's more common to rename to `_config_entry` if you want to highlight the parameter is intentionally unused.

Reply:
Thanks for the tip. Done in [bd3a964](https://github.com/home-assistant/core/pull/96620/commits/bd3a964989e682bf0329885baf5ab030d5d5dddf)